datasets/fornalab-formica-wp1-vegetation/src/dwc_mapping.Rmd

```diff
@@ -175,7 +179,7 @@ event <-
     parentEventID = .data$parent_event_id,
     eventDate = .data$Date_of_survey,
     samplesizeValue = 9,
-    sampleSizeUnit = "m2",
+    sampleSizeUnit = "square metre",
     habitat = "temperate forest",
     samplingProtocol = "vegetation survey",
     country = dplyr::recode(
```

Comment:
If a country is missing/unmapped, you will not throw an error here. 

There is a package to do exactly this; `countrycode`, it's very convenient if you ever need to map many countrycodes.

Reply:
These codes are specific to this project. We had 2 regions in Sweden en used SS (south Sweden) en CS (central Sweden).
But good to know!